test/Main.purs

```diff
@@ -79,6 +82,14 @@ guitarSuite =
         (const $ failure "hidden by barre expected")
         (GVAL.validate guitarHiddenByBarre)
 
+tenorGuitarSuite :: Free TestF Unit
```

Comment:
```
→ Suite: tenor guitar serialization
  ✓ Passed: write C chord
  ✓ Passed: write F chord
  ☠ Failed: read C chord because expected { barre: Nothing, fingering: [0,0,2,2,2,0], firstFretOffset: 0, name:
"A" }, got { barre: Nothing, fingering: [0,0,2,3], firstFretOffset: 0, name: "C" }
  ☠ Failed: read F chord because expected { barre: (Just { fretNumber: 1, stringNumber: 0 }), fingering: [-1,3,2,2,-1,-1], firstFretOffset: 0, name: "F" }, got { barre: Nothing, fingering: [0,2,3,0], firstFretOffset: 0, name: "F" }
→ Suite: bass guitar serialization
  ✓ Passed: write G chord
  ✓ Passed: read G chord
  ✓ Passed: read bad JSON
  ✓ Passed: reject bad finger position
  ✓ Passed: reject bad string count
  ✓ Passed: reject bad first fret offset

2 tests failed:

In "tenor guitar serialization":
  In "read F chord":
    Error: expected { barre: (Just { fretNumber: 1, stringNumber: 0 }), fingering: [-1,3,2,2,-1,-1], firstFretOffset: 0, name: "F" }, got { barre: Nothing, fingering: [0,2,3,0], firstFretOffset: 0, name: "F" }
    at Module.error (file:///home/jgart/chord-editor/output/Effect.Exception/foreign.js:6:10)
    at Module.failure (file:///home/jgart/chord-editor/output/Test.Unit/index.js:347:40)
    at file:///home/jgart/chord-editor/output/Test.Unit.Assert/index.js:45:34
    at file:///home/jgart/chord-editor/output/Data.Validation.Semigroup/index.js:23:24
    at file:///home/jgart/chord-editor/output/Test.Main/index.js:174:181
    at $tco_loop (file:///home/jgart/chord-editor/output/Control.Monad.Free/index.js:91:60)
    at toView (file:///home/jgart/chord-editor/output/Control.Monad.Free/index.js:105:23)
    at go (file:///home/jgart/chord-editor/output/Control.Monad.Free/index.js:286:17)
    at file:///home/jgart/chord-editor/output/Control.Semigroupoid/index.js:6:24
    at $tco_loop (file:///home/jgart/chord-editor/output/Control.Monad.Free/index.js:91:60)

In "tenor guitar serialization":
  In "read C chord":
    Error: expected { barre: Nothing, fingering: [0,0,2,2,2,0], firstFretOffset: 0, name: "A" }, got { barre: Nothing, fingering: [0,0,2,3], firstFretOffset: 0, name: "C" }
    at Module.error (file:///home/jgart/chord-editor/output/Effect.Exception/foreign.js:6:10)
    at Module.failure (file:///home/jgart/chord-editor/output/Test.Unit/index.js:347:40)
    at file:///home/jgart/chord-editor/output/Test.Unit.Assert/index.js:45:34
    at file:///home/jgart/chord-editor/output/Data.Validation.Semigroup/index.js:23:24
    at file:///home/jgart/chord-editor/output/Test.Main/index.js:173:185
    at $tco_loop (file:///home/jgart/chord-editor/output/Control.Monad.Free/index.js:91:60)
    at toView (file:///home/jgart/chord-editor/output/Control.Monad.Free/index.js:105:23)
    at go (file:///home/jgart/chord-editor/output/Control.Monad.Free/index.js:286:17)
    at file:///home/jgart/chord-editor/output/Control.Semigroupoid/index.js:6:24
    at $tco_loop (file:///home/jgart/chord-editor/output/Control.Monad.Free/index.js:91:60)

[error] Tests failed: exit code: 1
```

Reply:
@newlandsvalley 

I pushed my branch with the failing tests. Do you happen to know what I may be missing for the read tests to pass?

Reply:
Oops ignore, I pushed the passing tests for read now :)

Reply:
@newlandsvalley I pushed all the tests and they are all passing! :)

Reply:
Good news!